google/generativeai/text.py

```diff
@@ -215,6 +216,23 @@ def _generate_response(
     return Completion(_client=client, **response)
 
 
+def count_text_tokens(
+    model: model_types.AnyModelNameOptions,
+    prompt: str,
+    client: glm.TextServiceClient = None,
+):
+    base_model = models.get_base_model_name(model)
+
+    if client is None:
+        client = get_default_text_client()
+
+    result = client.count_text_tokens(
+        glm.CountTextTokensRequest(model=base_model, prompt={"text": prompt})
+    )
+
+    return type(result).to_dict(result)
```

Comment:
Can we get a TypedDict or something for this return type? Even a `dict[str...]` or something would do as an interim.